docs/fastdds/api_reference/rtps/common/Types/types_toc.rst

```diff
@@ -0,0 +1,20 @@
+.. _api_rtps_common_types:
+
+Types
+--------------------
+
+.. toctree::
+
+    /fastdds/api_reference/rtps/common/Types/BuiltinEndpointSet_t
```

Comment:
I believe `inline std::ostream& operator<<(std::ostream& output, const ProtocolVersion_t& pv)` is missing